src/shared/tpm2-util.c

```diff
@@ -4015,6 +4015,9 @@ int tpm2_policy_pcr(
                         ESYS_TR_NONE,
                         NULL,
                         pcr_selection);
+        if (rc == TPM2_RC_PCR_CHANGED)
+                return log_debug_errno(SYNTHETIC_ERRNO(EBUSY),
```

Comment:
hmm, please use a more exotic error code here. maybe EUCLEAN or so. 

who knows EBUSY might be something kernel raises on its own in some context, for example if the tpm device is blocked or so...

Reply:
EUCLEAN sounds good, changed